cubedash/templates/dataset.html

```diff
@@ -161,11 +161,11 @@ <h2>Raw Metadata</h2>
                 zoom: 3,
                 center: [-26.2756326, 134.9387844],
                 layers: [osm_carto_light, dataset_data],
-                zoomControl: false,
-                attributionControl: false,
+                zoomControl: true,
+                attributionControl: true,
                 scrollWheelZoom: false
             });
-            L.control.zoom({position: "bottomright"}).addTo(map);
+            // L.control.zoom({position: "bottomright"}).addTo(map);
```

Comment:
The attribution is currently being put in the page footer instead of each individual map, so that it's less noisy, see here: https://github.com/opendatacube/datacube-explorer/blob/0eced1737952ecbdc18294a628a782cde3508741/cubedash/templates/layout/base.html#L63
Do you think we need to change it?

The zoom control is being added manually so that it's right-aligned on the map instead of left-aligned. Because otherwise the alignment looks off against the rest of the page:

![image](https://user-images.githubusercontent.com/25688/75215128-df135400-57e3-11ea-83e2-fdf31257e4fb.png)

(I should have added comments to make these look less like an accident)